### PR TITLE
added the value of trust

### PR DESCRIPTION
**Pull request by pedroomtz17**
Isaac, are these changes ok?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,4 +13,10 @@ Copyright 1993,1998,2014 Toyoda Masashi (mtoyoda@acm.org)
...
\ No newline at end of file
+Competition is our world; top performance is our goal. For we want to create value on a permanent basis. “Quality First” is a core corporate value at Continental: Our products and services always meet the highest quality standards. We overcome obstacles and boundaries. And we never let up in any of these endeavors.
+
+## Trust
+
+<img src="https://www.incimages.com/uploaded_files/image/1920x1080/getty_475294806_2000133318188430258_173581.jpg" width="600" height="400">
+
+Trust is the basis for everything we do and the prerequisite for our success. We keep our promises. To all our employees and all stakeholders in Continental (such as customers, employees, investors, business partners, politicians and society). We are aware that trust is hard to gain and can be easily destroyed. That is why we prove at all times that the trust placed in us is worth it.
```
Comment (IsaacRayas) [inline README.md:22]: It's okay, just to compliment, you could add Continental's vision.

**Review by IsaacRayas — COMMENTED** [review]
It's okay, just to compliment, you could add Continental's vision.

**Review by IsaacRayas — CHANGES_REQUESTED** [review]
It's okay, just to compliment, you could add Continental's vision.

**Comment by pedroomtz17** [comment]
Changes made 👍

**Review by IsaacRayas — APPROVED** [review]
Excellent, well done

**Pull request merged**